releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,5 +1,14 @@
 ## Microsoft Extensions Azure App Configuration
 
+### v2.0.0-preview-009200001-1437 - July 10, 2019
+* Added `ConfigureRefresh` method to configure dynamic refresh for configuration settings
```

Comment:
>Added `ConfigureRefresh` method to configure dynamic refresh for configuration settings [](start = 2, length = 87)

I'm trying to put my self in the shoes of a developer reading these release notes, and I'm thinking that it would be more helpful if rather than treating these like disjointed addition and removal of APIs we wrote it something like

* Replaced Watch capability with on-demand refresh mechanism.
  * Watch has some bad things
  * Refresh solves these bad things, and allows you to refresh on demand
* `Watch` no longer available
* Added `ConfigureRefresh`
* Added `GetRefresher`

That's an abbreviated version, but I think the structure is more helpful because the release was more of a redesign/replacement rather than adding/removing things. What do you think?

Reply:
Agree with @jimmyca15. I was thinking about the same.
(I assume "bad things" are just placeholder :))